lib/ClimaLandSimulations/Project.toml

```diff
@@ -42,9 +43,12 @@ cuDNN = "02a925ec-e4fe-4b08-9a7e-0d78e3d38ccd"
 ArtifactWrappers = "0.2"
 Bonito = "3"
 CairoMakie = "0.11"
-ClimaComms = "0.5.6"
-ClimaParams = "0.10"
+ClimaComms = "0.5.6, 0.6"
+ClimaCore = "0.13.2, 0.14"
+ClimaDiagnostics = "0.2"
+ClimaParams = "0.10.2"
```

Comment:
Double check if you need this

Reply:
I think we should comeback to `ClimaLandSimulations` in another PR, it has a lot of things to adjust...